README.md

```diff
@@ -1,83 +1,169 @@
+data from Julia code directly or to manipulate it with packages of the Julia
+open-source package ecosystem, [GridapMakie.jl](https://github.com/gridap/GridapMakie.jl) is born. As a part of the Google Summer of
+Code 2021 program, GridapMakie adopts [Makie.jl](https://github.com/JuliaPlots/Makie.jl) as a second visualization back-end for
+Gridap.jl simulations. This package is thought as a built-in tool to assess the user in their FE calculations with a smoother workflow
+in a highly intuitive API.
 
-# Examples
+## Installation
 
-```@example README
-using Gridap, GridapMakie, Makie
+According to Makie's guidelines, it is enough to install one of its backends, e.g. GLMakie. Additionally, Gridap provides the plot objects
+to be visualized and `FileIO` allows to save the figures plotted.
 
-model = simplexify(CartesianDiscreteModel((-2pi,2pi,-pi,pi), (20,10)))
-scene = plot(model)
+```julia
+julia> ]
+pkg> add Gridap, GridapMakie, GLMakie, FileIO
 ```
 
-![](_readme/images/2d.png)
+## Examples
 
-```@example README
-data = GridapMakie.demo_data(spacedim=1, valuetype=Float64)
-scene = plot(data.u, data.model)
-```
+First things first, we shall be using the three packages as well as `FileIO`.
+We may as well create directories to store downloaded meshes and output files
 
-![](_readme/images/plot_1d_Scalar.png)
+````julia
+using Gridap, GridapMakie, GLMakie
+using FileIO
+mkdir("models")
+mkdir("images")
+````
 
-```@example README
-model = simplexify(CartesianDiscreteModel((-2pi,2pi,-pi,pi), (10,10)))
-V = TestFESpace(reffe=:Lagrangian, order=1, valuetype = Float64,
-    conformity=:H1, model=model)
-f(pt) = sin(pt[1])*cos(pt[2])
-u = interpolate(V, f)
-scene = mesh(u, model, color=:yellow)
-```
+### 2D Plots
 
-![](_readme/images/mesh_2d_Scalar.png)
+Then, let us consider a simple, 2D simplexified cartesian triangulation Ω
 
-```@example README
-model = simplexify(CartesianDiscreteModel((-2pi,2pi,-pi,pi), (10,10)))
-V = TestFESpace(reffe=:Lagrangian, order=1, valuetype = Float64,
-    conformity=:H1, model=model)
-f(pt) = sin(pt[1])*cos(pt[2])
-u = interpolate(V, f)
-scene = wireframe(u, model)
-```
+````julia
+domain = (0, 1, 0, 1)
+cell_nums = (10, 10)
+model = CartesianDiscreteModel(domain, cell_nums) |> simplexify
+Ω = Triangulation(model)
+````
 
-![](_readme/images/wireframe_2d_Scalar.png)
+The visualization of the vertices, edges, and faces of Ω can be achieved as follows
 
-```@example README
-model = CartesianDiscreteModel((-1,1,-1,1), (10,10))
-V = TestFESpace(reffe=:Lagrangian, order=1, valuetype = VectorValue{2, Float64},
-    conformity=:H1, model=model)
-f(pt) = VectorValue(-pt[2], pt[1])
-u = interpolate(V, f)
-scene = arrows(u, model, arrowsize=0.1, lengthscale=0.5)
-```
+````julia
+fig = plot(Ω)
+wireframe!(Ω, color=:black, linewidth=2)
+scatter!(Ω, marker=:star8, markersize=20, color=:blue)
+save("images/2d_Fig1.png", fig)
+````
 
-![](_readme/images/arrows_2d_Vec2d.png)
+![](_readme/images/2d_Fig1.png)
```

Comment:
@paurierap ... a final comment on my side with regards to the README.jl file. Can the images be down scaled? (e.g., reduce to 2/3 of current size) I think this would improve the reading experience.

Reply:
How do they look now?

Reply:
can they be horizontally centered? perhaps you might have to use html embedded within markdown for such purpose 

if you find it too complicated, let us leave the figures as they were at the beginning 

Thanks!

Reply:
There you have it!

Reply:
Great!